sdk/eventhub/azure-eventhub/samples/async_samples/authenticate_with_named_key_credential_async.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to demonstrate utilizing AzureNamedKeyCredential to authenticate with Event Hubs asynchronously.
+"""
+
+# pylint: disable=C0111
+
+import os
+import time
+from azure.core.credentials import AzureNamedKeyCredential
+from azure.eventhub.aio import EventHubProducerClient
+from azure.eventhub import EventData
+
+# Target namespace and hub must also be specified.
+FULLY_QUALIFIED_NAMESPACE = os.environ['EVENT_HUB_HOSTNAME']
+EVENTHUB_NAME = os.environ['EVENT_HUB_NAME']
+
+EVENTHUB_POLICY_NAME = os.environ['EVENT_HUB_POLICY_NAME']
+EVENTHUB_KEY = os.environ['EVENT_HUB_KEY']
```

Comment:
the env vars should be:  (other sample tests are failing :P)
```suggestion
EVENTHUB_POLICY_NAME = os.environ['EVENT_HUB_SAS_POLICY']
EVENTHUB_KEY = os.environ['EVENT_HUB_SAS_KEY']
```